src/aiodynamo/client.py

```diff
@@ -949,7 +949,7 @@ async def transact_write_items(
         items: Sequence[Union[Put, Update, Delete, ConditionCheck]],
         *,
         request_token: Optional[str] = None,
-    ) -> None:
+    ) -> Any:
```

Comment:
This function never explicitly returns. Why `Any`?

Reply:
Yeah to resolve mypy error `function does not return value` and it seems the default return type for a missing annotation is `Any` in this case?

Reply:
Ah okay, will take a look at https://github.com/python/mypy/pull/13219 🙏